src/DataTransfer/podDataCreator/domain/measurement/ranges.go

```diff
@@ -0,0 +1,42 @@
+package measurement
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+type Ranges struct {
+	safe    [2]int
+	warning [2]int
+}
+
+func NewRanges(safeRangeStr string, warningRangeStr string) Ranges {
+	safeRange := getRangesFromString(strings.ReplaceAll(safeRangeStr, " ", ""))
+	warningRange := getRangesFromString(strings.ReplaceAll(warningRangeStr, " ", ""))
+
+	return Ranges{safe: safeRange, warning: warningRange}
```

Comment:
en vez de hacer primero una variable y luego asignárselo a los valores del struct usa directamente la funcion dentro del struct
```suggestion
	return Ranges{
	    safe: getRangesFromString(strings.ReplaceAll(safeRangeStr, " ", "")),
	    warning: getRangesFromString(strings.ReplaceAll(warningRangeStr, " ", ""))
	}
```

Reply:
Prefiero que este separado para que no sea muy largo